apps/api/src/app/message-template/usecases/create-message-template/create-message-template.usecase.ts

```diff
@@ -20,7 +20,13 @@ export class CreateMessageTemplate {
     let item = await this.messageTemplateRepository.create({
       cta: command.cta,
       name: command.name,
-      variables: command.variables,
+      variables: command.variables?.filter((variable) => {
```

Comment:
Is it possible maybe to avoid duplication a and have a helper function to clean to prepare the variables?